portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenuWidget.testcase

```diff
@@ -78,6 +78,117 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-107833. Can not view Global Navigation Menus via Navigation Menu widget in normal site if the user without View permission."
+	@priority = "4"
+	test CannotViewGlobalNavigationMenuWithoutPermission {
+		task ("Add a navigation menu in Global site") {
```

Comment:
Don't we need to remove this Navigation Menu in a postcondition? The next tests are failing due to this.

Reply:
I missed this point. I will add a new commit, then resend to you.

Reply:
Thanks!